test/TokenTestUtils.js

```diff
@@ -404,17 +429,17 @@ async function setLongDecimalFeesTransferWithFees(token, ownerAccount, arbitrary
     let balance3 = await token.balanceOf(arbitraryAccount);
     assert.equal(balance3, 1000);
     let balanceFeeAccount = await token.balanceOf(feeAccount);
-    assert.isTrue(new BigNumber(balanceFeeAccount).minus(new BigNumber(initialBalanceFeeAccount)).isEqualTo(new BigNumber(feeAmount)));
+    assert.isTrue(balanceFeeAccount.sub(initialBalanceFeeAccount).cmp(newBigNumber(feeAmount))==0);
```

Comment:
I would suggest using equals here instead of cmp as well just to avoid having to write ==0 in each test that uses it.

Reply:
see note about regarding BN library and assert.equals.